lib/apiservers/engine/backends/container.go

```diff
@@ -1377,14 +1377,20 @@ func validateCreateConfig(config *types.ContainerCreateConfig) error {
 	config.HostConfig.Memory = memoryMB
 	log.Infof("Container memory: %d MB", config.HostConfig.Memory)
 
-	if config.NetworkingConfig == nil {
-		config.NetworkingConfig = &dnetwork.NetworkingConfig{}
-	}
-
 	if config.HostConfig == nil || config.Config == nil {
 		return BadRequestError("invalid config")
 	}
 
+	if config.NetworkingConfig == nil {
+		config.NetworkingConfig = &dnetwork.NetworkingConfig{}
+	} else {
+		// If NetworkConfig exists, set NetworkMode to the default endpoint network, assuming only one endpoint network as the default network during container create
+		for networkName := range config.NetworkingConfig.EndpointsConfig {
```

Comment:
So just to confirm again: we don't allow the the container to have more than one endpoint network upon creation, right? Additional user-defined networks can be added by "docker network connect" after "docker create".